src/main/java/com/google/sps/servlets/DrinkRatingServlet.java

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// Servlet to process users rating a drink from a coffee shop
+@WebServlet("/drink-rating")
+public class DrinkRatingServlet extends HttpServlet {
+
+  private final double maxRating = 5.0;
+  private List<String> knownDrinks = new ArrayList<>();
+  public Map<String, Integer> votes = createMap();
+
+  private Map<String, Integer> createMap() {
+    Map<String, Integer> map = new LinkedHashMap<>();
+
+    /* Initializes a map with 5 keys corresponding to the ratings and gives each
+     *  of the keys a value of 0 indicating how many people have made that rating so far
+     */
+    for (int i = 1; i <= 5; i++) map.put(Integer.toString(i), 0);
+    return map;
+  }
+
+  private double getAverageRating(Map<String, Integer> votes) {
+
+    int totalRatings = 0;
+    int totalVotes = 0;
+
+    for (int i = 1; i <= votes.size(); i++) {
+      String str = Integer.toString(i);
+      totalRatings = i * votes.get(str);
+      totalVotes += votes.get(str);
+    }
+
+    double average = (double) (totalRatings / totalVotes);
+    average = Math.round(average * 10) / 10.0;
+    return average;
```

Comment:
I thought the plan was to create a method with a name that clarifies we're rounding to one decimal place. Assume this will be done in a subsequent pull request?